updatecli/update_api_server_files.yaml

```diff
@@ -0,0 +1,79 @@
+name: "Get latest Kubernetes released version"
```

Comment:
maybe we can rename this file to something else, as it is fetching the source files from both `apiserver` and `apimachinery` modules.